Documentation/Blazorise.Docs/Pages/Blog/2022-08-31_CSSTipsForDevelopers/Index.md

```diff
@@ -0,0 +1,115 @@
+image-url: img/blog/2022-08-31/css.png
+image-text: Practical CSS Tips For Developers
+author-name: James Amattey
+author-image: james
+posted-on: August1 31st, 2022
+read-time: 3 min
+
+---
+
+Cascading Style Sheet, the beauty behind the web, friend to some, a devil to many but a necessary evil. Cascading Style Sheets, at their root, are intended to separate the style of a webpage from the content.
+
+Whether you are using Blazorise with [bootstrap](https://bootstrapdemo.blazorise.com/), [bulma](https://bulmademo.blazorise.com/) or [antdesign](https://antdesigndemo.blazorise.com/), CSS will be a huge part of your job. 
+
+Fortunately, here are some great CSS tips for developers.
+
+---
+
+## Center anything in three lines of code
+To center one `div` inside another, we make the containing `div` a flex container. Next, set align-items to center to perform centering on the block axis, and justify-content to center to perform centering on the inline axis.
+
+In the future, we may be able to center elements without needing to turn the parent into a flex container, as the Box Alignment properties used here are specified to apply to block layout too.
+
+The CSS `justify-content` property defines how the browser distributes space between and around content items along the main axis of a flex container, and the inline axis of a grid container.
+
+The CSS `justify-items` property defines the default justify-self for all items of the box, giving them all a default way of justifying each box along the appropriate axis.
+
+``` CSS
+.center {
+  width: 300px;
+  height: 400px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+```
+
+---
+
+## Resize An Image To Fit
+
+Images can be a tough customer. When you create a container for a div, you usually want an image to fit in without stretching. Luckily, we can achieve that with 4 lines of code. 
+
+Using a [Blazorise Card Component](/docs/components/card), we will demonstrate how we fit an image with 4 lines of code. 
+
+```HTML
+<Card Margin="Margin.Is4.OnY">
+    <CardImage Source="https://blazorise.com/img/gallery/7.jpg" Alt="Placeholder image">
+    </CardImage>
+    <CardBody>
+        <CardTitle Size="5">Card title</CardTitle>
+        <CardText>
+            Some quick example text to build on the card title and make up the bulk of the card's content.
+        </CardText>
+        <Button Color="Color.Primary">Button</Button>
+    </CardBody>
+</Card>
+```
+
+The object-fit CSS property sets how the content of a replaced element, such as an `img` should be resized to fit its container
+
+```CSS
+.cardimage {
+height: 100%;
+width: 100%;
+object-fit: contain
+}
+```
+
+---
+
+## Truncate Text With Ellipsis
+If you have a `TextEdit` (https://blazorise.com/docs/components/textedit) or `FieldLabel`(https://blazorise.com/docs/components/field) Component, you truncate the text content to limit the number of words so it does not expand beyond a certain limit. 
```

Comment:
Internal links must NOT contain the domain names. Instead of `https://blazorise.com/docs/components/textedit` it must be `docs/components/textedit`